### DISCUSSION TITLE
Missing plot in chia UI

### DISCUSSION THREAD

**Discussion by Tanwwir** [Chia-Network/chia-blockchain#4959] [Support]
Hello I recently did a k-34 plot (first plot ever). The logs stated it was finished the file displays in both my temp and final destination folder. However, the plot does not display on chia blockchain ui. Originally it did not say farming after completing at 100% so I restarted the chia blockchain application and now it does not show any plot and only displays the "add a plot button."

PC: AMD ryzen 7 5800x 8core
Ram: 64

**Comment by VictorLaszio** [comment]
Hi
Just check the following;
In the final destination folder does it say .plot  or  .plot.tmp
if it's .plot well then the file is probably OK

Then try click the ellipsis (the dots) beside the "Add a Plot" button and select "add  plot directory" there you can add the final destination folder.

See if that works.

**Comment by moshenskyDV** [comment]
@Tanwwir 100% on display it isn't finish yet. Probably you interrupted copying of final file to destination folder (as @VictorLaszio mentioned). Next time be patient and wait until it changes status to "mining".
PS: sorry about k34, I suppose it was long time waiting :(

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
